main.js

```diff
@@ -55,4 +55,140 @@ navAbout.addEventListener('click', () => {
   if (dropStatus === true) {
     closeDrop();
   }
-});
\ No newline at end of file
+});
+
+/* -------------------------------------------------- */
+/*                  DATABASE                     */
+/* -------------------------------------------------- */
+const projects = [
+  {
+    id: 1,
+    name: 'Figma project 1',
+    description:
+      'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Atque dolorum excepturi tempora suscipit velit asperiores est explicabo non cumque reprehenderit consectetur ut, minus nihil sed incidunt possimus quas deleniti maiores, tempore magnam rerum nulla fugiat.',
+    imageUrl: '/images/app_sceenshot.jpg',
+    technologies: ['html', 'Ruby on rails', 'Javascript'],
+    liveLink: 'http://nuri1977.github.io/bootsrap5-figma-01/',
+    sourceLink: 'https://github.com/Nuri1977/bootsrap5-figma-01',
+  },
+  {
+    id: 2,
+    name: 'Figma project 2',
+    description:
+      'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Atque dolorum excepturi tempora suscipit velit asperiores est explicabo non cumque reprehenderit consectetur ut, minus nihil sed incidunt possimus quas deleniti maiores, tempore magnam rerum nulla fugiat.',
+    imageUrl: '/images/app_sceenshot.jpg',
```

Comment:
- [ ] The image is broken, kindly fix it, I suggest you declare it as  `imageUrl: './images/app_sceenshot.jpg',` instead of  `imageUrl: '/images/app_sceenshot.jpg',`😉